code/modules/clothing/rogueclothes/feet.dm

```diff
@@ -78,70 +65,61 @@
 /obj/item/clothing/shoes/roguetown/simpleshoes/lord
 	name = "shoes"
 	desc = ""
-	gender = PLURAL
 	icon_state = "simpleshoe"
 	item_state = "simpleshoe"
 	resistance_flags = null
 	color = "#cbcac9"
 
-/obj/item/clothing/shoes/roguetown/gladiator
-	name = "leather sandals"
-	desc = ""
-	gender = PLURAL
-	icon_state = "gladiator"
-	item_state = "gladiator"
-	sewrepair = TRUE
 
-/obj/item/clothing/shoes/roguetown/sandals
-	name = "sandals"
-	desc = ""
-	gender = PLURAL
-	icon_state = "sandals"
-	item_state = "sandals"
-	sewrepair = TRUE
+// **ARMOURED** //
 
-/obj/item/clothing/shoes/roguetown/shalal
-	name = "babouche"
-	desc = "Simple cheap shoes."
-	gender = PLURAL
-	icon_state = "shalal"
-	item_state = "shalal"
-	sewrepair = TRUE
-	armor = list("blunt" = 25, "slash" = 20, "stab" = 25, "bullet" = 0, "laser" = 0,"energy" = 0, "bomb" = 0, "bio" = 0, "rad" = 0, "fire" = 0, "acid" = 0)
 
-/obj/item/clothing/shoes/roguetown/boots/armor
+/obj/item/clothing/shoes/roguetown/boots //This subtype is for armoured shoes
```

Comment:
How do you feel about /armoured or something instead?

Reply:
This decision was made before I decided to replace a lot of subtypes, I was afraid to mess with a lot of files and have this in PR review hell. I will turn boots into armour or armor or armoured as you see fit!

Reply:
Resolved! Changed from "boots" to "armor" in the same vein as other armors are done!